### PR TITLE
Update common packages to resolve https://www.npmjs.com/advisories/1674

### PR DESCRIPTION
**Pull request by tatyana-kostromskaya**
**Task name**: common npm packages: atrifacts-common, kubernetes-common-v2, securefiles-common, utility-common-v2

**Description**: Update versions of "azure-pipelines-tool-lib" and "azure-devops-node-api" to fix high vulnerability related to "underscore" package.
image

**Documentation changes required:** (Y/N) N

**Added unit tests:** (Y/N) N

**Attached related issue:** (Y/N) 

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [ ] Checked that applied changes work as expected

### CODE REVIEW

**Comment by anatolybolshakov** [comment]
@tatyana-kostromskaya could you please also make sure that these packages won't be picked up automatically for any of in-the-box tasks - until we explicitly bump it for task?

File: common-npm-packages/artifacts-common/package.json
Comment (anatolybolshakov) [inline common-npm-packages/artifacts-common/package.json:18]: Since there is a minor bump for node-api - could it make sense to bump minor number for common package also?

File: common-npm-packages/securefiles-common/package.json
Comment (anatolybolshakov) [inline common-npm-packages/securefiles-common/package.json:22]: The same also here

**Comment by tatyana-kostromskaya** [comment]
> @tatyana-kostromskaya could you please also make sure that these packages won't be picked up automatically for any of in-the-box tasks - until we explicitly bump it for task?

I checked all package.json and package-lock.json files, new versions won't be picked up automatically

**Review by anatolybolshakov — APPROVED** [review]

**Comment by anatolybolshakov** [comment]
@tatyana-kostromskaya these changes looks good, but  please don't merge PR till we have at least 2 approvals

**Pull request merged**
